### DISCUSSION TITLE
Can't Work Due to Constant Drive Activity

### DISCUSSION THREAD

**Discussion by tangierc** [CommandPost/FCPCafe#389] [Final Cut Pro Questions]
Hey folks. Checking to see if any of you have experienced this with FCP. I just started a new project, imported all of the media (about 13TB of ProRes 4444), created ProRes Proxies, and synced all the media with Sync-N-Link. I am still in organization mode. No editorial yet. I mention that because it's usually during editorial on the storyline when I start having performance issues. Here's where things get strange.

1. More often than not when I select a clip in FCP or go to another event, the drive starts working hard and I get the spinning beach ball - Optimized/original or proxy playback modes.
2. When I go to the desktop (Finder), another open app, or anywhere outside of FCP and then come back to FCP the drive works really hard and I get the spinning beachball, unable to continue working for a while until it's done.  Optimized/original or proxy playback modes.
3. I considered maybe FCP keeps trying to create waveform cache files so I scaled down all of my clips in icon view, expanded the window to see as many clips as possible at once to get FCP to draw the waveforms up front so I don't have to worry about it later. This didn't fix the problem. 

I've worked on much larger projects in FCP so I am not convinced yet that the size of the library is the issue. Here are project, app, and hardware specs below:

1. Final Cut Pro 10.7.1
2. MacBook Pro M2 Max w/64 GB of Ram
3. FCP library is on the MBPs internal SSD separate from the media on the external drive. Cache is in the library.
4. MacOS Sonoma 14.5
5. Provided [OWC Gemini ](https://eshop.macsales.com/shop/owc-gemini?utm_term=&utm_campaign=PMax:+External+drives+(smart+shopping)+US&utm_source=google&utm_medium=cpc&hsa_acc=9360794409&hsa_cam=17553315506&hsa_grp=&hsa_ad=&hsa_src=x&hsa_tgt=&hsa_kw=&hsa_mt=&hsa_net=adwords&hsa_ver=3&gad_source=1&gbraid=0AAAAAD_cuHTRqMEffj4tdamYGfChEtHc1&gclid=CjwKCAjw2dG1BhB4EiwA998cqPmjtyJTsVAXBsH3GyboBSf7RGFWAN8_TzcqTJOUcjE32SU-MkYYBRoCR4YQAvD_BwE)preconfigured and formatted to RAID 0, Mac OS Extended (Journaled) GUID partition map.
6. OWC RAID connected via Thunderbolt 3 (via the MBP's Thunderbolt 4 port)
7. I use an app called [Amphetamine](https://apps.apple.com/us/app/amphetamine/id937984704?mt=12) to keep the drive from sleeping. There is an issue, at least in Sonoma, whereas various external drives on the Mac constantly spin down with just a short amount of inactivity. The system preference to "Put hard disks to sleep when possible" doesn't work, nor did the OWC-recommended Terminal command called "[caffeinate](https://www.theapplegeek.co.uk/blog/caffeinate)".

**Comment by tangierc** [comment]
On second thought. Now that I put clips int he timeline, every clip that goes not he timeline causes the drive to work feverishly and I have to wait to do anything else.

**Comment by britim** [comment]
Hello Tangier,

Whenever I experienced this issue with FCP it was usually with a large project where the media was stored on a spinning drive.
Whenever I transfer the project and media (proxy and or original) to an SSD drive the project responsiveness would return to normal.

I think even with RAID drives FCP (perhaps because of the thumbnail and waveform generation) can really overwhelm the spinning drive's ability to quickly access the smaller data files that FCP needs to operate smoothly and an SSD can satisfy those demands.

Perhaps make a proxy version of the project that is small enough to put onto an SSD then see how the performance is.
This should work fine enough for the edit.

When you're done with your edit you might consider creating a separate Library based on the Final sequence and relinking the media (on the spinning drives) to that sequence and consolidating that media  to the same or another SSD drive so you can continue online and output smoothly.

That's been the workflow that works for me particular for longer features and docs that have 100s of clips.
Hope that helps.

Brian

**Comment by tangierc** [comment]
Thanks Brian. Yes it is on a a spinning drive - OWC Gemini in RAID 0. It has no issue whatsoever with the ProRes 4444 files in FCP; seeking, scrubbing, selecting clips, editing on the timeline, etc. However when I switch to the lower quality and smaller file ProRes proxies, the drive goes crazy and works really hard. I can't do anything without it beach balling and the drive cranking away...on smaller files! This is odd behavior. OWC did see an error on the drive from a drive report I sent them. I've reinstalled my Sonoma OS and a fresh copy of all my apps just to rule my system out.

I keep the FCP library and the cache files on an SSD and the drive is connected via Thunderbolt 3.  Unfortunately to get to the bottom of this I'll have to wipe and reformat the drive. 

Saving a separate "final" library is something I typically do.

**Reply by tangierc** [reply]
Some interesting updates. After Installing a fresh Sonoma macOS and Apple Pro Apps, wiping and formatting the RAID and restoring the footage to it I still get that same problem whereas the drive just locks up FCP with the spinning beachball when switching to proxies no matter what I do. 


I am using FCP 10.8.1 as of this writing. 

I also noticed that in proxy mode where I had no proxies made yet, the footage still showed in the browser and playback window. This shouldn't happen. The media should be offline since there were no proxies made. I confirmed that by turning on the proxy codec column so I could see whether or not there is a proxy showing in FCP's database. Additionally there is no "reveal proxy in Finder" in the contextual menu. So why is proxy only mode treating the footage like it's present and just instead playing the original media?

Secondly, a a lot of clips and triggering FCP to make proxies locks up FCP with a spinning beachball. Typically once the operation gets going, I can still click around and do other things in FCP and the transcode will pause until I stop doing things in FCP.

I am curious if you all experience long seek times when doing a Shift+F (reveal in browser) operation. I do that on a this RAID for this project and the RAID works really hard clanking away (with a beachball in FCP) before it can show me the clip in the browser.

**Comment by britim** [comment]
Yes this is consistent with what I experienced with FCP running on spinning drives.
-You try to click in the browser and you get the spinning beach ball for several minutes.
-You try to click into a keyword collection and you get the same.

The app generally becomes unusable putting your stomach in knots.

Somewhere around 10.4-.5 or so it seems that FCP just did not run as well when accessing large amounts of media running on spinning hard drives.
Can't say I know the machinations of this issue but have experienced across a few projects.
I think it might have something to do with the thumbnail files but not entirely sure.

Once I moved the projects to an SSD drive all is well.

You might be able to break the project down to smaller Events though that can potentially create other organizational issues. 

Consider trying the SSD route for now.

Perhaps Apple can be alerted that people still work with spinning drives and  RAIDs and that the app should have improved performance when using those media.

britim
